### DISCUSSION TITLE
Instructions tell users to start rails with `rails s`

### DISCUSSION THREAD

**Discussion by jasonfb** [shakacode/react_on_rails#1485] [Q&A] [answered]
Please note I am stuck getting the basic installation working so perhaps there is a gap in my understanding. please elucidate me if so.
see https://github.com/shakacode/react_on_rails/issues/1483 for my issue where I am stuck here.

however, I tried both `foreman start -f Procfile.dev-static` and also `rails s` and both produce the same results (see issue)

What I don't understand is if the webpack dev server is needed to be running in the background to make it work when compile is set to false (which the react_on_rails installer has just done in the prior step), why do these instructions tell the user to start the server with `rails s` and not `foreman start -f Procfile.dev-static` 

![Getting Started | Shakacode 2022-09-23 08-54-27](https://user-images.githubusercontent.com/59002/191965410-fbb07a61-330e-4cd6-bc5d-9b2ef80b3452.png)

**Comment by justin808** [comment, marked as answer]
Thanks! I opened a doc issue. Feel free to submit a PR if we don't beat you to it!

**Comment by justin808** [comment]
As I wrote in #1488:

> FYI -- if webpacker.yml setting for compile is true, then the current instructions are correct. However, that's not the recommended path, which is to either the webpack-dev-server via bin/webpacker-dev-server or run bin/webpacker --watch. These are setup by the Procfile examples.
